app-admin/passwordsafe/passwordsafe-1.09.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI-7 should work here.

Reply:
Hello @juippis ,

I figured out that EAPI-7 does not work without other changes. build fails with a "ld" error. I wanted to find out why in a later step, but with your comments it migth be easier for me now.
Will look at it, thanks for commenting!
Martin